docs/content/guides/rows/row-sorting.md

```diff
@@ -453,163 +1184,2251 @@ import 'handsontable/dist/handsontable.full.min.css';
+
+    multiColumnSortingPluginInstance.sort(
+      [
+        {
+          column: 0,
+          sortOrder: 'asc',
+        },
+        {
+          column: 1,
+          sortOrder: 'desc',
+        },
+      ],
+    );
+  };
+
+  return (
+    <>
+      <HotTable
+        ref={hotTableComponentRef}
+        data={[
+          {
+            brand: 'Jetpulse',
+            model: 'Racing Socks',
+            price: 30,
+            sellDate: '11/10/2023',
+            sellTime: '01:23',
+            inStock: false,
+          },
+          {
+            brand: 'Jetpulse',
+            model: 'HL Mountain Frame',
+            price: 1890.90,
+            sellDate: '03/05/2023',
+            sellTime: '11:27',
+            inStock: false,
+          },
+          {
+            brand: 'Jetpulse',
+            model: 'Cycling Cap',
+            price: 130.10,
+            sellDate: '27/03/2023',
+            sellTime: '03:17',
+            inStock: true,
+          },
+          {
+            brand: 'Chatterpoint',
+            model: 'Road Tire Tube',
+            price: 59,
+            sellDate: '28/08/2023',
+            sellTime: '08:01',
+            inStock: true,
+          },
+          {
+            brand: 'Chatterpoint',
+            model: 'HL Road Tire',
+            price: 279.99,
+            sellDate: '02/10/2023',
+            sellTime: '13:23',
+            inStock: true,
+          },
+        ]}
+        columns={[
+          {
+            title: 'Brand',
+            type: 'text',
+            data: 'brand',
+          },
+          {
+            title: 'Model',
+            type: 'text',
+            data: 'model',
+          },
+          {
+            title: 'Price',
+            type: 'numeric',
+            data: 'price',
+            numericFormat: {
+              pattern: '$ 0,0.00',
+              culture: 'en-US'
+            },
+            className: 'htLeft',
+          },
+          {
+            title: 'Date',
+            type: 'date',
+            data: 'sellDate',
+            className: 'htRight',
+          },
+          {
+            title: 'Time',
+            type: 'time',
+            data: 'sellTime',
+            className: 'htRight',
+          },
+          {
+            title: 'In stock',
+            type: 'checkbox',
+            data: 'inStock',
+            className: 'htCenter',
+          },
+        ]}
+        multiColumnSorting={true}
+        height="auto"
+        width="auto"
+        licenseKey="non-commercial-and-evaluation"
+      />
+      <div className="controls">
+        <button onClick={sort}>Sort</button>
+      </div>
+    </>
+  );
+}
+
+/* start:skip-in-preview */
+ReactDOM.render(<HandsontableComponent />, document.getElementById('example8'));
+/* end:skip-in-preview */
+```
+
+:::
+
+:::
+
+### Exclude rows from sorting
+
+You can exclude any number of top or bottom rows from sorting.
```

Comment:
Let's give it a business context - you may want to do that in case of fixed rows at the top (usually used as column names then), or at the bottom (used for summary calculations). I would put the demo there with both frozen rows at the top and the bottom - must scroll!